feed/api-updates.rss

```diff
@@ -4,7 +4,14 @@
     <title>Upvest API updates</title>
     <link>https://docs.upvest.co/news</link>
     <description>Updates of the API release notes</description>
-    <pubDate>Thu, 26 Sep 2024 13:00:00 +0000</pubDate>
+    <pubDate>Tue, 01 Oct 2024 13:00:00 +0000</pubDate>
+    <item>
+      <title>v1.37.0 Corporate action notifications</title>
+      <description>Operation models: TOL, BYOL, OMNIBUS\\nChanges: ...</description>
```

Comment:
Would you like to write a short summary of the changes?
Also for the new line use `&lt;br/&gt;` (updated the guide)